addons/a11y/src/components/ColorBlindness.tsx

```diff
@@ -72,15 +72,15 @@ export class ColorBlindness extends Component<ColorBlindnessProps, ColorBlindnes
     const { filter, expanded } = this.state;
 
     let colorList = [
-      'protanopia',
-      'protanomaly',
-      'deuteranopia',
-      'deuteranomaly',
-      'tritanopia',
-      'tritanomaly',
-      'achromatopsia',
-      'achromatomaly',
-      'mono',
+      'Protanopia',
+      'Protanomaly',
+      'Deuteranopia',
+      'Deuteranomaly',
+      'Tritanopia',
+      'Tritanomaly',
+      'Achromatopsia',
+      'Achromatomaly',
+      'Mono',
```

Comment:
Just upcase the first letter instead?

Reply:
Actually, like this idea better because it has the least impact so I made that change instead.